public/src/main/java/com/revenuecat/purchases/models/StoreTransaction.kt

```diff
@@ -92,7 +92,12 @@ data class StoreTransaction(
      * In Google, this will be calculated from the basePlanId and offerId
      * Null for restored transactions and purchases initiated outside of the app.
      */
-    val subscriptionOptionId: String?
+    val subscriptionOptionId: String?,
+
+    /**
+     * The proration_mode string to be sent to the backend.
```

Comment:
Hmm this seems not that useful for developers? Not sure if they would need to get it from the SDK for something... Maybe something like:
"The prorationMode used to perform the upgrade/downgrade of this purchase. Null if it was not an upgrade/downgrade or if the purchase was restored. This is not available for Amazon purchases."

Reply:
Makes sense, I wasn't aware whether this is exposed to the app developers. Added the doc and additionally added the fact that it's the name of GoogleProrationMode enum